contracts/kernel/Kernel.sol

```diff
@@ -44,6 +49,7 @@ contract Kernel is IKernel, KernelStorage, Petrifiable, IsContract, VaultRecover
 
         acl.initialize(_permissionsCreator);
 
+        setDepositable(true);
```

Comment:
I think depositing to the kernel should be disabled by default, given that there is no reason to send funds to the kernel for now. I think it is good that the `KernelProxy` keeps the functionality (in case the Kernel having ETH makes sense at some point), but it should be disabled IMO.

Reply:
My main reason for enabling it was because Kernels are assigned ENS names through our initial templates, and so I could see users sending ETH directly to that name.

However, thinking about it, it's less confusing if the kernel reverts on receiving ETH directly than requiring a user to manually use the escape hatch functionality to transfer the funds to the vault. In the future, a different ENS name could be used for a finance or funding app, e.g. `finance.<dao name>.aragonid.eth`.